### ISSUE TITLE
Update for 1.7.10 (please)

### ISSUE THREAD

**Issue by iLLHunter** [watkins577/Mods#4]
Please update for compatibility with MC1.7.10

I do not see any license or details about someone else updating your source. Is that possible?

**Comment by watkins577** [comment]
I have sent the source to @artemrussia, I am not sure on how much he is planning to update it, but he's the guy to go to :+1:

**Comment by iLLHunter** [comment]
Since he has not done any updating it only shows this repo under "Repositories contributed to" on his page, so I can not contact him via a new issue, it will only make a new issue here. 
Does he read these issues? Can you suggest a way to contact him?
@artemrussia if you read these issues, would you share your plans? Will you have time to update the mod? Would you like help? Is there an ETA on a 1.7.10 update?

**Comment by nskartem** [comment]
@iLLHunter yeah, i has time to update it, and i'll set up forge 1.7.10 tomorrow. Can you test new version when it will be ready, please?

**Comment by iLLHunter** [comment]
@artemrussia I would be happy to test it out

**Comment by nskartem** [comment]
@iLLHunter well, i will begin it tomorrow, because now 11 PM, and i wanna sleep.

**Comment by iLLHunter** [comment]
alright. great. I'll look for new version
